server.go

```diff
@@ -698,8 +698,7 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 		return nil, err
 	}
 
-	queryBandwidth := func(edge *channeldb.ChannelEdgeInfo) lnwire.MilliSatoshi {
-		cid := lnwire.NewChanIDFromOutPoint(&edge.ChannelPoint)
+	queryBandwidth := func(cid lnwire.ChannelID) lnwire.MilliSatoshi {
```

Comment:
Non-blocking, just a thought. Since we are doing refactoring here, we might as well move this method back to the `routing` package since the `routing` package is already using the `htlcswitch`. This could improve the overall code structure by getting rid of the cyclic/jumpy definition of the function, but def involves more work tho.

Reply:
Great idea!